cypress/support/pages/registrate.pageObject.js

```diff
@@ -0,0 +1,13 @@
+import PageObject from '../PageObject';
+
+class registratePageObject extends PageObject {
+    url = '/user/register';
+
+regNewUser (username, email, password) {
```

Comment:
Method names should be camelCase. 'regNewUser' should be 'regNewUser'. Also, it's a good practice to start method names with a verb, so 'registerNewUser' would be better.